### DISCUSSION TITLE
Custom fields in "worklog report"

### DISCUSSION THREAD

**Discussion by templarvii** [shridhar-tl/jira-assistant#294] [Q&A]
Our Jira tickets have a custom field "Job Number".
Is it possible to add them to the "Worklog Report"?

**Comment by shridhar-tl** [comment]
Hi @templarvii , Apologies for delayed response, I have implemented this feature a long back, however it is not yet published as extension. It would be available as part of JA v2.49. Once your extension gets updated, you should be able to use it from extension.

However, it is already available in web version. You can right away switch to web version seamlessly and start using it until your extension gets updated.

Closing this discussion assuming your questions is answered and issue is resolved. Please feel free to reopen it or start a new discussion if you have any further queries.
